app/presenters/article_template_presenter.rb

```diff
@@ -1,4 +1,7 @@
 class ArticleTemplatePresenter < TemplatePresenter
   include SharedComponents::Strategy
   include SharedComponents::Teaser
+
+  delegate :meta_description, to: :article
+  delegate :meta_title, to: :article
```

Comment:
We can change to `delegate :meta_title, :meta_description, to: :article`

Reply:
Good point 👍